src/ConfigDefinition.php

```diff
@@ -16,9 +16,10 @@ protected function getParametersDefinition(): ArrayNodeDefinition
         /** @noinspection NullPointerExceptionInspection */
         $parametersNode
             ->children()
-                ->scalarNode('bucket')->end()
+                ->scalarNode('bucket')->end() // this is only for compatibility with legacy configs and is ignored
                 ->scalarNode('#token')->isRequired()->cannotBeEmpty()->end()
                 ->scalarNode('url')->isRequired()->cannotBeEmpty()->end()
+                ->booleanNode('fullSync')->defaultFalse()->end()
```

Comment:
co se stane, když bude `fullSync` a `incremental` zároveň? jak jem psal výš, možná tohle znamená ~`add`/`update`/`replace`?

Reply:
nic, incremental se odignoruje, ale tohle se mi celkem libi, zkusim to udelat jako
mode: `recreate`, `replace`, `update`, uvidim co to hodi